core/blocksigner/enclave.go

```diff
@@ -42,6 +43,12 @@ func (ec EnclaveClient) Sign(ctx context.Context, pk ed25519.PublicKey, bh *lega
 		client.BaseURL = tup[0]
 		client.AccessToken = tup[1]
 
+		// If the provided access token is just a password with no
+		// user, use a default basic auth username of 'chaincore'.
+		if !strings.Contains(client.AccessToken, ":") {
+			client.AccessToken = "chaincore:" + client.AccessToken
+		}
```

Comment:
Good question. AFAIK, Chain Core has always expected a user:password access token. If the input is just a string, it would only set the username in the basic auth request.

This change was motivated by my own confusion and [this TODO](https://github.com/chain/chain/blob/main/cmd/cored/main.go#L403-L404). Requiring it to have a colon sounds just as good to me.